packages/ia-userlist-settings/src/ia-user-list-settings.ts

```diff
@@ -40,6 +40,8 @@ export class IAUserListSettings extends LitElement {
 
   private async saveListDetails(event: Event) {
     event.preventDefault();
+    const submitButton = (event.target as HTMLElement)?.querySelector('button#save-list-settings');
+    submitButton?.setAttribute('disabled', 'true');
```

Comment:
QUESTION: If one re-opens the "Add to List" modal later on the same Details page (for example, after the user has submitted a list request which completes successfully), will the `disabled` attribute be cleared?

Reply:
hi yes, that's working as expected.